systemd/ceph-exporter@.service.in

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Ceph cluster exporter daemon
+PartOf=ceph-exporter.target
+After=network-online.target local-fs.target time-sync.target
+Before=remote-fs-pre.target ceph-exporter.target
+Wants=network-online.target local-fs.target time-sync.target remote-fs-pre.target ceph-exporter.target
+
+[Service]
+Environment=CLUSTER=ceph
+EnvironmentFile=-@SYSTEMD_ENV_FILE@
```

Comment:
what env variables does `ceph-exporters` read? again, i think you were replicating the existing pattern. but i wanted to make sure the service unit does exactly what we want, but not more than what is necessary.

Reply:
I believe that users can use this setting to change their ceph cluster's name.

In [this issue](https://tracker.ceph.com/issues/39139), for example,  a user complains that they cannot override the `CLUSTER` variable (though it turned out that there was no bug in Ceph in this case.)

Reply:
ack. thanks for your explanations. but `--cluster` has been deprecated. again, i think we should at least consider dropping this in the new code. see https://docs.ceph.com/en/latest/rados/configuration/common/#naming-clusters-deprecated

Reply:
dropped in #55560

Reply:
Thank you for your explanation for and sharing the document.
I removed both `Environment` and `EnvironmentFile` settings, and removed `--cluster` option from the command in `ExecStart`.